src/3d/qgs3dmapscene.h

```diff
@@ -194,10 +194,10 @@ class _3D_EXPORT Qgs3DMapScene : public QObject
      * Returns a map of 3D map scenes (by name) open in the QGIS application.
      *
      * \note Only available from the QGIS desktop application.
-     *
+     * \deprecated since QGIS 3.36, use QgisAppInterface::mapCanvases3D() instead.
      * \since QGIS 3.30
      */
-    static QMap< QString, Qgs3DMapScene * > openScenes();
+    Q_DECL_DEPRECATED static QMap< QString, Qgs3DMapScene * > openScenes() SIP_DEPRECATED;
```

Comment:
@nyalldawson do you want to deprecate + keep this, or is it okay for you if we remove it altogether?